cake/build-settings.cake

```diff
@@ -152,17 +136,10 @@ public class BuildSettings
         if (IsRunningOnWindows)
         {
             StandardRunnerTests.Add(Net60WindowsFormsTest);
-
-            if (!IsRunningOnAppVeyor)
-            {
-                StandardRunnerTests.Add(NetCore21X86Test);
-                StandardRunnerTests.Add(NetCore31X86Test);
```

Comment:
I see it is remove here, but why would a .net core test be limited to Windows?
The `Net462X86Test` is in the standard list.

Reply:
Currently, we only support X86 under Windows. AgentProcess.cs makes a check and has a //TODO to remove the limitation. I removed this because I couldn't run it on my own machine and forgot to come back to it. There should really be a check for dotnet x86 being installed in the script, just as there is in our code.

Reply:
Added a property IsDotNetX86Installed and used it here. Property currently checks for Windows, which we can remove at some future point. I'll push and see if this works on AppVeyor as well. Thanks for pointing this out.